PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/appservice/component/input/AzureComboBox.java

```diff
@@ -57,9 +57,15 @@
     private AzureComboBoxEditor inputEditor;
 
     public AzureComboBox() {
+        this(true);
+    }
+
+    public AzureComboBox(boolean refresh) {
         super();
         this.init();
-        this.refreshItems();
+        if (refresh) {
+            this.refreshItems();
+        }
```

Comment:
I also fixed this problem in my PR;
```
    public AzureComboBox() {
        super();
        this.init();
        DefaultLoader.getIdeHelper().invokeLater(this::refreshItems);
    }
```

Reply:
For some component, it may not need to fresh in constructor, so create an overloading for it